aait/web/group-1/components/profile/Card.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Image from 'next/image';
+import { Blog } from '@/types/Blog';
+import { BiSolidTime } from 'react-icons/bi';
+import Link from 'next/link';
+
+interface PropsInterface {
+  post: Blog;
+}
+
+const Card: React.FC<PropsInterface> = ({ post }) => {
+  const dateString = post.createdAt;
+  const date = new Date(dateString);
+
+  const options: Intl.DateTimeFormatOptions = { year: 'numeric', month: 'short', day: 'numeric' };
+  const dateFormatted = date.toLocaleDateString('en-US', options);
```

Comment:
for the date formatter, it's fine as it is but if this date format is used anywhere else it's better to use a date formatter library like moment.js and add the date formatter file another place like under "/utils" folder or somewhere common to make it accessible for other components.